include/seastar/core/smp.hh

```diff
@@ -67,6 +67,11 @@ struct smp_service_group_config {
     ///
     /// Will be adjusted upwards to allow at least one request per non-local shard.
     unsigned max_nonlocal_requests = 0;
+    /// The group smp group name
```

Comment:
I think there is some typo in this sentence? maybe something liek  "An optional name for this group"?

Reply:
done